csaf_2.1/prose/edit/src/tests-02-optional.md

```diff
@@ -1111,7 +1111,7 @@ The relevant path for this test is:
               "selections": [
                 {
                   "name": "Technical Impact",
-                  "namespace": "an-yet-unknown-or-maybe-private-namespace",
+                  "namespace": "some-yet-unknown-or-maybe-private-namespace",
```

Comment:
Yes please, let us keep a self-describing test value. Thanks. (not using and-...)